{{ cookiecutter.project_slug }}/flows/index.yml

```diff
@@ -1,9 +1,5 @@
 !Flow
 pods:
-  crafter:
-    uses: pods/craft.yml
-    parallel: $JINA_PARALLEL
-    read_only: true
   encoder:
```

Comment:
I am not sure, but my understanding is, that we should do the same pre-processing for index and query time, at least for images. My best guess is, we also need a `uses_before` here.

Reply:
Is not explicitly needed to have the same crafter for indexing and querying, but it would be the most common usecase is to have the same preprocessing, therefore I do not see it being removed

Reply:
sure, retained it.